build_scripts/ubuntu_sim_common_deps.sh

```diff
@@ -16,38 +16,19 @@ trap "exit" INT TERM; trap "kill 0" EXIT; sudo -v || exit $?; sleep 1; while tru
 echo "We must first remove modemmanager"
 sudo apt-get remove modemmanager -y
 
-# Ninja build system
-ninja_dir=$HOME/ninja
-echo "Installing Ninja to: $ninja_dir."
-if [ -d "$ninja_dir" ]
-then
-    echo " Ninja already installed."
-else
-    pushd .
-    mkdir -p $ninja_dir
-    cd $ninja_dir
-    wget https://github.com/martine/ninja/releases/download/v1.6.0/ninja-linux.zip
-    unzip ninja-linux.zip
-    rm ninja-linux.zip
-    exportline="export PATH=$ninja_dir:\$PATH"
-    if grep -Fxq "$exportline" ~/.profile; then echo " Ninja already in path" ; else echo $exportline >> ~/.profile; fi
-    . ~/.profile
-    popd
-fi
 
 # Common dependencies
 echo "Installing common dependencies"
-sudo add-apt-repository ppa:george-edison55/cmake-3.x -y
 sudo apt-get update -y
-sudo apt-get install python-argparse git git-core zip python-empy python-toml python-numpy qtcreator cmake build-essential genromfs -y
+sudo apt-get install python-argparse git git-core zip python-empy python-toml python-numpy qtcreator cmake build-essential genromfs ninja-build -y
 # Required python packages
 sudo apt-get install python-dev -y
 sudo apt-get install python-pip -y
 sudo -H pip install --upgrade pip
 sudo -H pip install pandas jinja2
-pip install pyserial
+sudo -H pip install pyserial
```

Comment:
- combine all python installs here with pip?
 - git-core is a legacy debian thing that can be dropped

Reply:
Thanks, I've removed git-core. I've put pyserial along with jinja2. 
I've also grouped the apt-gets as it says in the text - all the python ones are no together. 
OK to merge?